src/utils/menuConditions.ts

```diff
@@ -0,0 +1,54 @@
+import { Role } from '@sendbird/chat';
+import type { UserMessage } from '@sendbird/chat/message';
+import type { GroupChannel } from '@sendbird/chat/groupChannel';
+import type { OpenChannel } from '@sendbird/chat/openChannel';
+import { SendableMessageType, isFailedMessage, isPendingMessage, isSentMessage, isUserMessage } from '.';
+
+export const isReplyTypeMessageEnabled = ({ channel, message }) => {
+  if (isFailedMessage(message) || isPendingMessage(message)) {
+    return false;
+  }
+  if (!channel?.isGroupChannel?.() || channel?.isEphemeral) {
+    return false;
+  }
+  const isBroadcast = (channel as GroupChannel)?.isBroadcast;
+  const isOperator = channel?.myRole === Role.OPERATOR;
+  return isBroadcast ? isOperator : true;
```

Comment:
```suggestion
  const isBroadcast = (channel as GroupChannel)?.isBroadcast;
  const isOperator = channel?.myRole === Role.OPERATOR;
  if (isBroadcast) return isOperator; 
  return true;
```
How about this? This ternary operator made me think twice 😅